crates/swc_ecma_codegen/tests/test262-min/cb625ce2970fe52a.js

```diff
@@ -1 +1 @@
-(function(){'use\x20strict';with(a);})
+(function(){'use strict';with(a);})
```

Comment:
I think this is a bomb, although I don't expect someone to write code like `'use\x20strict'` unless they are testing...